lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -476,6 +476,42 @@ describe('AlfrescoViewerComponent', () => {
         //
     });
 
+    describe('originalMimeType', () => {
+        it('should set originalMimeType based on nodeData content', async () => {
+            const defaultNode: Node = {
+                id: '123',
+                name: 'Mock Node',
+                nodeType: 'cm:content',
+                isFolder: false,
+                isFile: true,
+                modifiedAt: new Date(),
+                modifiedByUser: { id: 'user123', displayName: 'John Doe' },
+                createdAt: new Date(),
+                createdByUser: { id: 'user456', displayName: 'Jane Doe' },
+                isLocked: false,
+                parentId: 'parent123',
+                isLink: false,
+                isFavorite: false,
+                content: {
+                    mimeType: 'application/msWord',
+                    mimeTypeName: 'Doc Document',
+                    sizeInBytes: 0
+                },
+                aspectNames: ['cm:auditable', 'cm:versionable'],
+                properties: { customProperty: 'customValue' },
+                allowableOperations: ['update', 'delete', 'create-child'],
+                path: { name: '/Sites/Document Library/Mock Node', isComplete: true },
+                permissions: {},
+                definition: {}
+            };
+
+            spyOn(component['viewUtilService'], 'getViewerType').and.returnValue('unknown');
+
+            await component['setUpNodeFile'](defaultNode);
```

Comment:
Again - it is not proper way to test because you should not refer to private things using []. I know this is private function but you should test it by checking in which public place that private function is called and call that public thing here. 

I checked it and that setUpNodeFile function is called in following places: 
- in onNodeUpdated function but it is again private so checking further it is called in ngOnInit which is public and can be called in tests 
- in setupNode function which is again private but is called in ngOnChanges which you can call in tests because it is public. 
Ideally you should test it for all paths so calling through ngOnInit and through ngOnChanges (you can write 2 separate tests)